collectors/VmonCollector.py

```diff
@@ -34,13 +43,11 @@ def collect(self):
             for service in fetched_data['value']:
                 service_name = service['key']
                 state = service['value']['state']
-                if state != "STARTED":
-                    health = "None"
+                if state == "STOPPED":
+                    health_state = "STOPPED"
                 else:
-                    health = service['value']['health']
-                metric_value = 1
-                if health != 'HEALTHY':
-                    metric_value = 0
-                g.add_metric(labels=[service_name, health, state, vc], value=metric_value)
+                    health_state = service['value']['health']
```

Comment:
please change health_state to service_health, that improves `health_states[service_health]` readability.